eng/build.py

```diff
@@ -0,0 +1,178 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+"""Build automation scripts"""
+
+import os
+import subprocess
+import sys
+from typing import List

```

Comment:
Note that this script does not depend on any external libraries and does not need to run in a virtual environment.